packages/at_client/lib/src/client/at_client_impl.dart

```diff
@@ -903,107 +857,6 @@ class AtClientImpl implements AtClient, AtSignChangeListener {
     return _preference;
   }
 
-  @override
-  Future<String?> notifyChange(NotificationParams notificationParams) async {
```

Comment:
As this method was deprecated a long time ago, marked it deprecated here also, moved it to the end of this file, and made it a simple wrapper around NotificationService.notify()